src/main/flight/pid.c

```diff
@@ -524,17 +550,40 @@ static void detectAndSetCrashRecovery(
     }
 }
 
-static void handleItermRotation()
+static void rotateVector(float v[3], float rotation[3]) 
 {
-    // rotate old I to the new coordinate system
-    const float gyroToAngle = dT * RAD;
-    for (int i = FD_ROLL; i <= FD_YAW; i++) {
+    // rotate v around rotation vector rotation
+    // rotation in radians, all elements must be small
+    for (int i = 0; i < 3; i++) {
```

Comment:
this previously was only active for pitch/roll from what I can see, is that accurate? same again w/ XYZ_AXIS_COUNT

Reply:
I think FD_YAW is 2 and  (i <= 2) == (i < 3)?

Reply:
Oh I see what you mean. Yeah I've now enabled it on yaw as well.